packages/synapse-react-client/src/components/AccessRequirement/EntityActionsRequired.tsx

```diff
@@ -15,11 +16,15 @@ export const EntityActionsRequired: React.FunctionComponent<
   const { data: actionRequiredList } = useGetEntityActionsRequired(entityId)
   const actions = actionRequiredList?.actions
 
+  const allCompleteAndIncompleteActions = useTrackTransientListItems(
+    actionRequiredList?.actions,
+  )
```

Comment:
Use custom hook to track actions even as they are completed. This way, the action card does not immediately disappear after the user completes it.